sdcm/nemesis.py

```diff
@@ -2159,23 +2175,40 @@ def get_ks_with_few_tables(keyspace_table):
         result = self.target_node.run_nodetool('clearsnapshot')
         self.log.debug(result)
 
-    # NOTE: '2022.1.rc0' is set in advance, not guaranteed to match when appears
-    # NOTE: current variant of the toppartitions feature is supported since 4.5 OSS
-    @scylla_versions(("4.5.rc1", None), ("2022.1.rc0", None))
+    # NOTE: '2023.1.rc0' is set in advance, not guaranteed to match when appears
+    @scylla_versions(("4.6.rc0", None), ("2023.1.rc0", None))
     def disrupt_show_toppartitions(self):
+        # NOTE: new API is supported only starting with Scylla 4.6
+        #       In Scylla 4.5 it exists, but is disabled.
+        self.log.info("DEBUG: disrupt_show_toppartitions , allow_new_api=True")
```

Comment:
if this a debug print ? why it's on info level ?

Reply:
Good catch. Fixed.